Sampling of dataset with frequency parameter returns error

### What happened?

Following error is returned when setting the `frequency` parameter in the stream config:
```
AssertionError: (<class 'numpy.timedelta64'>, np.timedelta64(21600000,'ms'))
[14] > /users/jkuehner/CODE/WeatherGenerator/.venv/lib/python3.12/site-packages/anemoi/utils/dates.py(188)as_timedelta()
-> assert isinstance(frequency, str), (type(frequency), frequency)
```

### What are the steps to reproduce the bug?

Set the following parameter in the stream config:
`frequency : 06:00:00`

### Hedgedoc link to logs and more information. This ticket is public, do not attach files directly.

_No response_

--- a/src/weathergen/datasets/data_reader_anemoi.py
+++ b/src/weathergen/datasets/data_reader_anemoi.py
@@ -17,7 +17,6 @@ from anemoi.datasets.data import MissingDateError
 from anemoi.datasets.data.dataset import Dataset
 from numpy.typing import NDArray
 
-from weathergen.common.config import parse_timedelta
 from weathergen.datasets.data_reader_base import (
     DataReaderTimestep,
     ReaderData,
@@ -65,7 +64,7 @@ class DataReaderAnemoi(DataReaderTimestep):
 
         kwargs = {}
         if "frequency" in stream_info:
-            kwargs["frequency"] = parse_timedelta(stream_info["frequency"])
+            kwargs["frequency"] = stream_info["frequency"]
         if "subsampling_rate" in stream_info:
             name = stream_info["name"]
             _logger.warning(

